src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -144,9 +147,15 @@
     <Compile Include="System\Data\SqlTypes\SQLString.cs" />
     <Compile Include="System\Data\SqlTypes\SQLUtility.cs" />
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
-	<Compile Include="$(CommonPath)\System\NotImplemented.cs"/>
+    <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
+    <Compile Include="Interop\LocaleInterop.cs" />
+    <Compile Include="Interop\ILocaleApiHelper.cs" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="Interop\LocaleWindowsApiHelper.cs" />
```

Comment:
Shouldn't these implementations be named as LocaleApiHelper.Windows.cs and LocaleApiHelper.Unix.cs to fit our naming conventions?